### DISCUSSION TITLE
Allow labels and areas to be applied to Zones from the Zones page

### DISCUSSION THREAD

**Discussion by gadgetchnnel** [home-assistant/frontend#20651] [Configuration]
Labels and areas can be added to Zones (useful for grouping zones by location), but these can only be applied by finding them in the Entities list and adding them from there:

![image](https://github.com/home-assistant/frontend/assets/2099542/f50e169e-ebf7-4d60-9130-c065d69e5eb2)

It would be useful if this could be done directly from the Zones page when editing or creating a zone so it's not necessary to switch between the Zones page and Entities list.

**Comment by sam2332** [comment]
This feature seems like something that could have helped me!
i just entered ~50 zones and i really would love to categorize and label them!
building automations with triggers for like 25 zones for my job suck!

**Comment by RUGMJ** [comment]
I wouldn't have know it was at all possible without this, would be great to have it in the main ui.
